Taskfile.yml

```diff
@@ -45,10 +45,6 @@ tasks:
         sh: gopass show cogito/test_commit_sha
       COGITO_TEST_OAUTH_TOKEN:
         sh: gopass show cogito/test_oauth_token
-      COGITO_TEST_READ_ONLY_COMMIT_SHA:
```

Comment:
so there are no tests using these variables?

Reply:
Not as far I am aware.
Here the list does not include them:
https://github.com/Pix4D/cogito/blob/04e9eb5dd4689591010f91488a739c5b3e14a6ba/help/testhelper.go#L160
and quick search in the repo does not find them anywhere except in this file.
Also, all e2e tests are passing without them.

Reply:
ok